solidity/test/TestPhasedEscrow.js

```diff
@@ -476,6 +509,69 @@ describe("StakerRewardsBeneficiary", () => {
     })
   })
 })
+
+describe("BeaconRewards to PhasedEscrow transfer", async () => {
+  const owner = accounts[0]
+  const operators = [accounts[1], accounts[2]]
+
+  const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+  const totalRewards = web3.utils.toBN(19800000).mul(tokenDecimalMultiplier)
+
+  before(async() => {
+    let contracts = await initContracts(
+      contract.fromArtifact('TokenStaking'),
+      contract.fromArtifact('KeepRandomBeaconService'),
+      contract.fromArtifact('KeepRandomBeaconServiceImplV1'),
+      contract.fromArtifact('KeepRandomBeaconOperatorBeaconRewardsStub')
+    )
+
+    token = contracts.token
+    stakingContract = contracts.stakingContract
+    operatorContract = contracts.operatorContract
```

Comment:
Would this work?
```suggestion
    const {token, stakingContract, operatorContract} = await initContracts(
      contract.fromArtifact('TokenStaking'),
      contract.fromArtifact('KeepRandomBeaconService'),
      contract.fromArtifact('KeepRandomBeaconServiceImplV1'),
      contract.fromArtifact('KeepRandomBeaconOperatorBeaconRewardsStub')
    )
```

Reply:
Unfortunately no - we need to have `token` and `operatorContract` declared on a global level so that they are accessible from `it`s.